set.mm

```diff
@@ -674802,6 +674806,554 @@ fixed reference functional determined by this vector (corresponding to
       TYAYAWNUAUBBCDYBYD $.
   $}
 
+
+  $c PrimRoots $.
+  $( Define the class of primitive roots.  (Contributed by metakunt,
+     25-Apr-2025.) $)
+  cprimroots $a class PrimRoots $.
+
+  ${
+    $d a b k l r $.
+    $( A r-th primitive root is a root of unity such that the exponent divides
+       r.  (Contributed by metakunt, 25-Apr-2025.) $)
```

Comment:
The HTML version of the comment _should_ look better if you math-escape the symbols:
```
A ` r ` -th primitive root is a root of unity such that the exponent divides ` r ` .
```